autosklearn/estimators.py

```diff
@@ -232,8 +232,13 @@ def __init__(
 
             Not all keys returned by scikit-learn are supported yet.
 
+        performance_over_time\_ : pandas.core.frame.DataFrame
+            A ``DataFrame`` containing the models performance over time data. Can be
+            used for plotting directly. Please refer to the example
+            :ref:`Train and Test Inputs <sphx_glr_examples_40_advanced_example_pandas_train_test.py>`_.
+
```

Comment:
Hopefully this is the last change but the `_` at the end had to be removed as well, it's used to indicate external links with `html` links where as `refs` don't need them.